frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -1080,6 +1080,16 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
                 </dd>
               </>
             )}
+            {
+              <>
+                <dt>{t('public~Control Plane Topology')}</dt>
+                <dd>
+                  {window.SERVER_FLAGS.controlPlaneTopology !== ''
```

Comment:
so I dont suppose that the `controlPlaneTopology` wont be set on the cluster, since its not a required infrastructure config field.

Reply:
Removing for now